tests/PortingAssistant.Client.UnitTests/UploaderTest.cs

```diff
@@ -31,5 +39,77 @@ public void Upload_Empty_File_Returns_Success_Status()
             Assert.IsTrue(actualSuccessStatus);
         }
 
+        [Test]
+        public void File_Line_Map_Updated_On_Upload()
+        {
+            var handlerMock = new Mock<HttpMessageHandler>(MockBehavior.Strict);
+            handlerMock
+               .Protected()
+               .Setup<Task<HttpResponseMessage>>(
+                  "SendAsync",
+                  ItExpr.IsAny<HttpRequestMessage>(),
+                  ItExpr.IsAny<CancellationToken>()
+               )
+               .ReturnsAsync(new HttpResponseMessage()
+               {
+                   StatusCode = HttpStatusCode.OK,
+               })
+               .Verifiable();
+
+            // use real http client with mocked handler here
+            var httpClient = new HttpClient(handlerMock.Object)
+            {
+                BaseAddress = new Uri("https://8q2itpfg51.execute-api.us-east-1.amazonaws.com/beta"),
+            };
```

Comment:
I see we use this endpoint in other tests. I'm assuming it's not public?

Reply:
Converted to localhost because I'm mocking the API call anyway. Thanks!